Behavior/BehaviorDelegate.swift

```diff
@@ -50,9 +48,17 @@ class BehaviorDelegate: SpeziAppDelegate {
             }
 
             if HKHealthStore.isHealthDataAvailable() {
-                healthKit
+                HealthKit {
+                    CollectSample(
+                        HKQuantityType(.stepCount),
+                        deliverySetting: .background(.afterAuthorizationAndApplicationWillLaunch)
+                    )
+                    CollectSample(
+                        HKQuantityType(.activeEnergyBurned),
+                        deliverySetting: .anchorQuery(.afterAuthorizationAndApplicationWillLaunch)
+                    )
+                }
```

Comment:
Please either move this section down to the existing healthKit computed property to ensure that you don't need to add the swiftlint disable command. The current `healthKit` computed property is not used.
```suggestion
                healthKit
```

Reply:
Please be sure that you remove the elements here and only inject the computed property as shown in the comment above. You can even easily commit the suggests with the buttons below the code suggestion.